src/scenes/ObservationsRouter/schedule/ObservationSubzoneSelector.tsx

```diff
@@ -78,7 +80,16 @@ const ObservationSubzoneSelector = ({ onChangeSelectedSubzones, plantingSite }:
               <Box sx={{ display: 'inline-block' }} key={_index}>
                 <Checkbox
                   id={`observation-subzone-${zone.id}`}
-                  label={subzone.name}
+                  label={
+                    subzone.totalPlants ? (
+                      subzone.name
+                    ) : (
+                      <Box>
+                        <Icon name='warning' style={{ margin: '4px 4px 4px 0', verticalAlign: 'bottom' }} />
```

Comment:
We need to upload the "black" warning icon

Reply:
Yea, that will come in a later PR, still waiting for it to be added to Zeroheight